src/ansys/dpf/post/selection.py

```diff
@@ -580,6 +580,36 @@ def select_nodes_of_elements(
             _WfNames.scoping, op.outputs.mesh_scoping_as_scoping
         )
 
```

Comment:
This is a new method. Before, requesting an elemental scoping with a nodal input yielded an error. If this was blocked intentionally, please let me know.